ansys/mapdl/core/_commands/apdl/parameter_definition.py

```diff
@@ -279,14 +279,27 @@ def get(
         command = f"*GET,{par},{entity},{entnum},{item1},{it1num},{item2},{it2num}"
         return self.run(command, **kwargs)
 
-    def inquire(self, func):
+    def inquire(self, strarray='', func='', arg1='', arg2=''):
         """Returns system information.
 
+        By default, with no arguments, it returns the working directory.
+
+        >>> mapdl.inquire()
+        C:\\Users\\gayuso\\AppData\\Local\\Temp\\ansys_nynvxsaooh
```

Comment:
Avoid using personally attributable examples.

```suggestion
        C:\\Users\\user\\AppData\\Local\\Temp\\ansys_nynvxsaooh
```